src/xtgeo/xyz/_xyz_roxapi.py

```diff
@@ -257,19 +356,16 @@ def export_xyz_roxapi(
     rox.safe_close()
 
 
-def _roxapi_export_xyz_hpicks(
-    self, rox, name, category, stype, realisation, attributes
-):  # pragma: no cover
-    """
-    Export/store as RMS horizon picks; this is only valid if points belong to wells
-    """
-    # need to think on design!
-    raise NotImplementedError
-
-
 def _roxapi_export_xyz_viafile(
-    self, rox, name, category, stype, pfilter, realisation, attributes
-):  # pragma: no cover
+    self: xtgeo.Points | xtgeo.Polygons,
+    rox: RoxUtils,
```

Comment:
This might just be a preference of my own, but you might consider just importing `Points` and `Polygons` directly from their modules and type as `Points | Polygons`. IMO I think we should get away from doing a bulk `import xtgeo` in every file and prefer precise imports unless otherwise necessary. But, I certainly don't want to impose my preferences!

Reply:
Yes, I tried to do that since mypy complains about them.. but I got a circular dependency error if I try to import them from their modules. So I threw in the towel 😄 but I will give it a second shot!

Reply:
With some aid from @janbjorge I got both mypy and the circular dependency to be happy by importing the points/polygons module instead of the class inside it!